games-fps/redeclipse/files/redeclipse

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+cd /usr/share/redeclipse
```

Comment:
File system operations can be a bit expensive. Rather than using `cd` here, use a path var.

BASE_DIR=/usr/share/redeclipse